monero-wallet-generator.html.asc

```diff
@@ -1,17 +0,0 @@
------BEGIN PGP SIGNATURE-----
-Version: GnuPG v1
-
-iQIcBAABAgAGBQJYlaa5AAoJEGhvB0VNbO/DP90QAJSHlHFtU4Gau4bgF4ZHTID7
-Vc0kvu0p3B3v9n/+zO02YKjYGHp1F0f2lyA+ZAnUW8BXR5QTy0QqqmfQDK4ilCQT
-2x7Xjcs30Ol90p1A2hIzqr+iY5qTnvqMriYpD39GUIFyqGOUsJj6+zP3mpTzgMLW
-U742K/whTFcqUdLWxzIsv41kflvdnpbphHsycANLwY/7owfVoZm0gZNwzTNB4IQ5
-dS+i2cp4GATZGqH5dxYBkKq7xlqN1/+GRx9bhvIa6fanI99ldm9ajmh7Ehd24Khl
-0lR/ngXk8HKWyRlNtVuUZ47i9lkUibygRVtL2HZ2ML+MUZrNRH3Hk5WJp/FyjQ8Y
-LFt/DVCBAf7o+Ohjrl/w5/SI4baY2B5YZhOKD5pWMlRfgZZp+aIWDDAwgYEUrZN6
-/gnfTrOLIWgsm+eZjMvVUopJrVDrinTkS5d03F8Yy9DH2P1qcF102dzHnK0bAwxR
-6BAwnCqYYsIZpKVeq/JUTC6OtQt3LImQzhbATCTAGEoSNs7zwQ600r1QkyzAJwFv
-FmJM4gTX09lISbQZKDPu0DAO967TTK8JQqNtGZ2ibFVVBoQ8posxJjnf2drjB5nD
-hMHc0vD9nE2NoTp+t/scOZw/avSGfvuSPmcBlpkWJqqZZG7o8RGFKt89qZlcIRQ8
-jt7ihU/WJxDiA5Qtwu2R
-=xteP
------END PGP SIGNATURE-----
```

Comment:
Please do not remove this

Reply:
My bad! I was worried to keep your signature on my repo. i was thinking that it's confusing if my commit is with your signature.